### DISCUSSION TITLE
How do I expose security onion using nginx proxy manager?

### DISCUSSION THREAD

**Discussion by samtheruby** [Security-Onion-Solutions/securityonion#10700] [Unsupported Versions]
Hi I'm starting out with security onion and I wanted to expose the management console to my domain using nginx proxy manager. It works with just  one click for just about any service except for security onion. In the installer is there anything I need to add or change to expose it? I tried connecting it to a proxy in the setup but that did not work either, I got the error curl (56): Received HTTP code 400 from proxy after connect. 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/53793412/270555a1-a25d-4310-9f02-d9c4b35971a0)
How would I properly add my nginx reverse proxy to security onion?

**Comment by sohaibaumer** [comment]
To expose the Security Onion management console using Nginx Proxy Manager:
1.	Install and set up Nginx Proxy Manager on your server.
2.	Access the Nginx Proxy Manager web interface.
3.	Create a new proxy host for Security Onion:
      o	Enter the IP address of your Security Onion server.
      o	Specify the port number used by the Security Onion management console.
      o	Enter the domain or subdomain you want to use to access Security Onion.
      o	Enable SSL if you have an SSL certificate.
4.	Configure DNS to ensure your domain/subdomain points to the Nginx Proxy Manager server.
5.	Save and apply the changes in Nginx Proxy Manager.
6.	Test access by visiting the configured domain/subdomain in your web browser.

Doing this you can expose the Security Onion management console using Nginx Proxy Manager. Remember to configure DNS, firewall rules, and SSL certificates as needed.

**Reply by samtheruby** [reply]
I tried this obviously, these steps work for all other services but it does not work for security onion.
